README.md

```diff
@@ -16,7 +16,12 @@ the square brackets (for multiple-choice questions).
 What are constructor functions and the `new` keyword? What is a prototype? Describe an example of when we would use a constructor function versus a prototype.
 
 Your Answer:
-```text
+```
+Constructor function is a special function that is used when we'll be creating multiple objects that are similar and have most of the same properties. Constructor function is basically a template for an object.
+
+When we call "new Constructor()" it creates a prototype based off the constructor function and it saves memory since those properties aren't all individually stored within each object, they share with and from the constructor.
+
+If we wanted every person in WDI-8 to be an object we could create a constructor function with properties that all students share (name, age, height, etc...) and then create prototypes for each student.
```

Comment:
:+1:  this is good, but be a careful about what you are calling prototype. the prototype is what the constructor function helps build. but each student would be _instances_ of the constructor, and _inherit_ the prototype from the student constructor